src/ir_Mitsubishi.cpp

```diff
@@ -632,12 +636,12 @@ uint8_t IRMitsubishiAC::convertFan(const stdAc::fanspeed_t speed) {
 /// @return The native equivalent of the enum.
 uint8_t IRMitsubishiAC::convertSwingV(const stdAc::swingv_t position) {
   switch (position) {
-    case stdAc::swingv_t::kHighest: return kMitsubishiAcVaneAutoMove - 6;
-    case stdAc::swingv_t::kHigh:    return kMitsubishiAcVaneAutoMove - 5;
-    case stdAc::swingv_t::kMiddle:  return kMitsubishiAcVaneAutoMove - 4;
-    case stdAc::swingv_t::kLow:     return kMitsubishiAcVaneAutoMove - 3;
-    case stdAc::swingv_t::kLowest:  return kMitsubishiAcVaneAutoMove - 2;
-    case stdAc::swingv_t::kAuto:    return kMitsubishiAcVaneAutoMove;
+    case stdAc::swingv_t::kHighest: return kMitsubishiAcVaneHighest;
+    case stdAc::swingv_t::kHigh:    return kMitsubishiAcVaneHigh;
+    case stdAc::swingv_t::kMiddle:  return kMitsubishiAcVaneMiddle;
+    case stdAc::swingv_t::kLow:     return kMitsubishiAcVaneLow;
+    case stdAc::swingv_t::kLowest:  return kMitsubishiAcVaneLowest;
+    case stdAc::swingv_t::kAuto:    return kMitsubishiAcVaneSwing;
     default:                        return kMitsubishiAcVaneAuto;
```

Comment:
Auto Vs swing here is confusing, can we have a comment describing why for any confused future reader that might see this as a bug?

Reply:
Good point. Heaps of comments and links to this PR and relevant Issue added.